course/07_Conditions.md

```diff
@@ -68,6 +68,17 @@ if (myVar) {
 ```
 This is useful for checking if the variable has an actual value.
 
+If you want to check if a something is not equal, or if a boolean variable is false, you can use a NOT gate. This is done simply b putting a `!` before the condition.
```

Comment:
Thank you for that contribution. I'd only fix the typo in the `by` word, than I can merge this PR